lib/spot-checker.js

```diff
@@ -112,9 +112,9 @@ export default class SpotChecker {
       stats.statusCode = res.status
 
       if (res.ok) {
-        resetTimeout()
         const reader = await CarBlockIterator.fromIterable(res.body)
         for await (const block of reader) {
+          resetTimeout()
           stats.byteLength += block.bytes.length
```

Comment:
For consistency of results, let's mirror https://github.com/CheckerNetwork/spark-checker/blob/15cc96d06bce48173459e9874f0d90cb9540e187/lib/spark.js as closely as possible:

1. Reset timeout before request is made (already the case)
2. Reset timeout when Response is ok (un-remove that line)
3. Reset timeout on each block (move line to bottom)